modules/gsuite_enabled/main.tf

```diff
@@ -70,6 +83,28 @@ resource "gsuite_group_member" "api_s_account_api_sa_group_member" {
   group = "${var.api_sa_group}"
   email = "${local.api_s_account}"
   role  = "MEMBER"
+}
 
-  depends_on = ["google_project_service.project_services"]
+module "project-factory" {
+  source              = "../core_project_factory"
+  random_project_id   = "${var.random_project_id}"
+  org_id              = "${var.org_id}"
+  domain = "${var.domain}"
+  name                = "${var.name}"
+  shared_vpc          = "${var.shared_vpc}"
+  billing_account     = "${var.billing_account}"
+  folder_id           = "${var.folder_id}"
+  group_name          = "${var.create_group ? "${gsuite_group.group.name}" : local.group_name}"
```

Comment:
The quotation and interpolation of `"${gsuite_group.group.name}"` looks to be redundant; since we're already in an interpolation expression we should be able to replace that with `gsuite_group.group.name`.